core/ui/compose/designsystem/src/main/kotlin/app/k9mail/core/ui/compose/designsystem/atom/text/TextDisplayMedium.kt

```diff
@@ -39,3 +46,43 @@ fun TextDisplayMedium(
         style = MainTheme.typography.displayMedium,
     )
 }
+
+@Composable
+fun AutoResizeableTextDisplayMedium(
```

Comment:
Please rename to `TextDisplayMediumAutoResize`.

Reply:
Done!